apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,108 +18,176 @@ export function InstructorCourseAdminSettings({
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is the official title of the course, as given in the course catalog.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="institution">Institution</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="institution"
+                  name="institution"
+                  value="${resLocals.institution.short_name} (${resLocals.institution.long_name})"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is your academic institution (e.g., "University of Illinois").
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="display_timezone">Timezone</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="display_timezone"
+                  name="display_timezone"
+                  value="${resLocals.course.display_timezone}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The allowable timezones are from the tz database. It's best to use a city-based
+                  timezone that has the same times as you.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="path">Path</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="path"
+                  name="path"
+                  value="${resLocals.course.path}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The path where course files can be found.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="repository">Repository</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="repository"
+                  name="repository"
+                  value="${resLocals.course.repository}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The Github repository that can be used to sync course files.
+                </small>
+              </div>
+            </form>
+            <form name="add-configuration-form" class="mb-3 mx-4" method="POST">
+              <div class="form-group">
+                <label for="configuration">Configuration</label>
+                <div
+                  class="form-control"
+                  id="configuration"
+                  name="configuration"
+                  value="infoCourse.json"
+                >
```

Comment:
This is a little funky: `div` elements shouldn't have a `value` attribute, and it's also a bit weird to be presenting the link styled as an input. Maybe replace the input + description with test that says `<a href="...">Edit course configuration</a> in <code>infoCourse.json</code>.`, where the `<a>` is a link to the edit page.

Reply:
That makes sense. I have updated this to remove it from an input and now just displaying it on it's own. I also removed the `Configuration` label as that part was also strange, especially now that there is no input box. Do you think it would be better if the `Edit course configuration` was a button instead of a link? I think the link looks fine but I'm not sure if it provides enough clarity that you should go there to edit the json file.

Here is what it looks like after the current changes:
![Screenshot 2024-02-14 at 4 45 16 PM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/a325101b-5f51-4500-adf6-66d43fb04fef)